.env-sample

```diff
@@ -0,0 +1,3 @@
+SNYK_TOKEN=replaceme
```

Comment:
Using a default value in `docker-compose-tests` means the `.env-sample` can be removed